_includes/navbar_brand.html

```diff
@@ -0,0 +1,4 @@
+<a class='navbar-brand' href="/">
+   <img width='300px' src='{{ "/assets/img/logo.svg" | prepend:site.baseurl }}')} />
```

Comment:
```suggestion
   <img width='300' src='{{ "/assets/img/logo.svg" | prepend:site.baseurl }}')} />
```

`px` are not defined in the `img` `width` and `height` attributes.